specs/protocol/isthmus/derivation.md

```diff
@@ -17,11 +24,261 @@ The Isthumus hardfork activation block contains the following transactions, in t
+jq -r ".bytecode.object" packages/contracts-bedrock/forge-artifacts/L1Block.sol/L1Block.json
+```
+
+This transaction MUST deploy a contract with the following code hash
+`TODO`.
+
+## GasPriceOracle deployment
+
+The `GasPriceOracle` contract is also upgraded to support the Isthmus operator fee feature.
+
+A deposit transaction is derived with the following attributes:
+
+- `from`: `0x4210000000000000000000000000000000000004`
+- `to`: `null`
+- `mint`: `0`
+- `value`: `0`
+- `gasLimit`: `1,625,000`
+- `data`: `0x60806040523480156100105...` ([full bytecode](../../static/bytecode/isthmus-gas-price-oracle-deployment.txt))
+- `sourceHash`: `0xfc70b48424763fa3fab9844253b4f8d508f91eb1f7cb11a247c9baec0afb8035`,
+  computed with the "Upgrade-deposited" type, with `intent = "Isthmus: Gas Price Oracle Deployment"
+
+This results in the Isthmus GasPriceOracle contract being deployed to `0x93e57A196454CB919193fa9946f14943cf733845`, to verify:
+
+```bash
+cast compute-address --nonce=0 0x4210000000000000000000000000000000000003
+Computed Address: 0xFf256497D61dcd71a9e9Ff43967C13fdE1F72D12
+```
+
+Verify `sourceHash`:
+
+```bash
+cast keccak $(cast concat-hex 0x0000000000000000000000000000000000000000000000000000000000000002 $(cast keccak "Isthmus: Gas Price Oracle Deployment"))
+# 0xfc70b48424763fa3fab9844253b4f8d508f91eb1f7cb11a247c9baec0afb8035
+```
+
+Verify `data`:
+
+```bash
+git checkout TODO
+make build-contracts
+jq -r ".bytecode.object" packages/contracts-bedrock/forge-artifacts/GasPriceOracle.sol/GasPriceOracle.json
+```
+
+This transaction MUST deploy a contract with the following code hash
+`TODO`.
+
+## Operator fee vault deployment
+
+A new `OperatorFeeVault` contract has been created to recieve the operator fees. The contract is created with
+the following arguments:
+
+- Recipient address: The base fee vault
+- Min withdrawal amount: 0
+- Withdrawal network: L2
+
+A deposit transaction is derived with the following attributes:
+
+- `from`: `0x4210000000000000000000000000000000000005`
+- `to`: `null`
+- `mint`: `0`
+- `value`: `0`
+- `gasLimit`: `500,000`
+- `data`: `0x60806040523480156100105...` ([full bytecode](../../static/bytecode/isthmus-operator-fee-deployment.txt))
+- `sourceHash`: `0x107a570d3db75e6110817eb024f09f3172657e920634111ce9875d08a16daa96`,
+  computed with the "Upgrade-deposited" type, with `intent = "Isthmus: Operator Fee Vault Deployment"
+
+This results in the Isthmus GasPriceOracle contract being deployed to `0x4fa2Be8cd41504037F1838BcE3bCC93bC68Ff537`, to verify:
+
+```bash
+cast compute-address --nonce=0 0x4210000000000000000000000000000000000003
+Computed Address: 0x4fa2Be8cd41504037F1838BcE3bCC93bC68Ff537
+```
+
+Verify `sourceHash`:
+
+```bash
+cast keccak $(cast concat-hex 0x0000000000000000000000000000000000000000000000000000000000000002 $(cast keccak "Isthmus: Operator Fee Vault Deployment"))
+# 0x107a570d3db75e6110817eb024f09f3172657e920634111ce9875d08a16daa96
+```
+
+Verify `data`:
+
+```bash
+git checkout TODO
```

Comment:
`TODO`

Reply:
It need to be a commit with a release tag

Reply:
Same